string bin_id
string object_id
string frame_id
bool mask_only
---
bool success
string message
string bin_id
sensor_msgs/PointCloud2 points
sensor_msgs/Image mask
sensor_msgs/Image mask_all
string object_id